go.mod

```diff
@@ -20,3 +20,5 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 	gotest.tools/v3 v3.0.3
 )
+
+replace github.com/compose-spec/godotenv => github.com/ulyssessouza/godotenv v1.4.1-0.20211116100545-655269e25657
```

Comment:
Shouldn’t we make a new release in `compose-spec/godotenv` instead?

Reply:
Or at least not point to your fork, now that the upstream PR has been merged?

Reply:
I assume this was a test PR to check the fix on godotenv side has no impact on compose-go
Maybe flag this as "do not merge" to avoid confusion?

Reply:
Well, to be fair it’s in a Draft state, but I received a request for review, so I thought I would do it as if the PR were ready.